packages/ui/src/proposals/components/ProposalFilters/SelectProposer.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Meta, Story } from '@storybook/react'
+import React, { useState } from 'react'
+
+import { SelectMember } from '@/memberships/components/SelectMember'
+import { MembershipContextProvider } from '@/memberships/providers/membership/provider'
+import { Member } from '@/memberships/types'
+import { MockApolloProvider } from '@/mocks/components/storybook/MockApolloProvider'
+
+import { SelectProposer } from './SelectProposer'
+
+export default {
+  title: 'Proposals/SelectProposer',
+  component: SelectProposer,
+  argTypes: {
+    onChange: { action: 'changed' },
+  },
+} as Meta
+
+export const Default: Story = ({ onChange }) => {
+  const [value, setValue] = useState<Member | null>(null)
+
+  return (
+    <MockApolloProvider>
+      <MembershipContextProvider>
+        <SelectMember onChange={onChange} /> {/* TODO remove */}
```

Comment:
TODO found ;) What we should do with that?

ps.: AFAIR the `<SelectMember />` looks OK when wrapped in the Input component (like in the forms).

Reply:
Sorry about that :facepalm:
I actually meant to remove it from the story before committing (I only added it to make sure I wasn't breaking `<SelectMember />` while editing the select component).

I will create an issue to add  `<SelectMember />` in a separate story though.